_maps/map_files/NSV_Triumph/triumph-04-deck4.dmm

```diff
@@ -28493,15 +28497,15 @@
 /area/security/security_bathroom)
 "uYv" = (
 /obj/structure/table/rack/shelf/steel,
-/obj/item/gun/projectile/automatic/l6_saw,
-/obj/item/gun/projectile/automatic/battlerifle,
-/obj/item/gun/projectile/automatic/battlerifle,
 /obj/effect/floor_decal/borderfloorblack{
 	dir = 10
 	},
 /obj/effect/floor_decal/corner/green/border{
 	dir = 10
 	},
+/obj/item/rig_module/mounted/energy_blade,
```

Comment:
you are aware these are the ninja rig energy blades right

Reply:
What ninjas? :honk:
But ye on a serious note, I don't really see a problem with using Ninja stuff as they're quite unsupported in the code atm